CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ and adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 ## [1.2.1] - 2021-03-26
 ### Added
 - Added `April21` to `ApiVersion` [#149](https://github.com/Shopify/shopify-node-api/pull/149)
+- Added `isActive()` method to `Session` class to check if session is active [#153](https://github.com/Shopify/shopify-node-api/pull/153)
```

Comment:
I might add to the changelog that certain things will now use `SessionInterface` instead of just `Session`, particularly where `Session` used to be the expected return type or argument type.